test/IRGen/musttail_fail.swift

```diff
@@ -0,0 +1,23 @@
+// RUN: %not --crash %target-swift-frontend -primary-file %s -parse-stdlib -module-name Swift -emit-ir
+
+class Klass {}
+
+// NOTE: We expect this to fail. This feature will be implimented in a future patch.
+// CHECK: musttail call swiftcc %Ts5KlassC* @"$ss3fooys5KlassCACF14mustTailReturnL_yA2CF"(
```

Comment:
I would just remove the CHECK since you aren't using it. I would also change the NOTE to be more like:

"We expect this test to fail since $INSERT_EXPLANATION_OF_EPILOGUE_BLOCK_PROBLEM. Support for more complicated CFGs will be implemented in a subsequent patch. We just want to make sure that LLVM asserts here as we expect it to without that support."